src/libcollections/vec.rs

```diff
@@ -1301,6 +1301,41 @@ impl<T: Clone> Vec<T> {
     }
 }
 
+impl<T: Default> Vec<T> {
+    /// Resizes the `Vec` in-place so that `len()` is equal to `new_len`.
+    ///
+    /// If `new_len` is greater than `len()`, the `Vec` is extended by the
+    /// difference, with each additional slot filled with `Default::default()`.
+    /// If `new_len` is less than `len()`, the `Vec` is simply truncated.
```

Comment:
Linking to our methods is good, we do that in other places. What does 'cloning' have to do with this method? Seems like it should say something along the lines of: "If you'd like to specify your own value if the `Vec` is extended, use `Vec::resize`".